docs/conf.py

```diff
@@ -70,7 +70,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = 'en'
```

Comment:
Explicitly specifying a version appears to be necessary in more recent versions of Sphinx.